ecommerce/extensions/api/v2/views/publication.py

```diff
@@ -42,3 +44,22 @@ def _save_and_publish(self, data, course_id=None):
                 content = serializer.data
                 content['message'] = message if message else None
                 return Response(content, status=status.HTTP_201_CREATED if created else status.HTTP_200_OK)
+
+
+class EdevateAtomicPublicationView(generics.CreateAPIView, generics.UpdateAPIView):
+    """Attempt to save and publish a Course and associated products.
+
+    If either fails, the entire operation is rolled back. This keeps Otto and the LMS in sync.
```

Comment:
probably better to change docstring to mention what is different from the normal AtomicPublicationView